CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# [0.20.0](https://github.com/dashevo/dapi-grpc/compare/v0.19.0...v0.20.0) (2021-07-08)
+
+
+### Features
+
+* add additional information to `GetTransactionResponse` ([#118](https://github.com/dashevo/dapi-grpc/issues/118), [#120](https://github.com/dashevo/dapi-grpc/issues/120))
+* add `metadata` to `WaitForStateTransitionResultResponse` ([#115](https://github.com/dashevo/dapi-grpc/issues/115))
+* rename `core_chain_locked_height` field ([#115](https://github.com/dashevo/dapi-grpc/issues/114))
+* add an additional set of fields for proofs  ([#115](https://github.com/dashevo/dapi-grpc/issues/112))
```

Comment:
Please combine them. It’s one single feature - add metadata and additional info to platform proofs